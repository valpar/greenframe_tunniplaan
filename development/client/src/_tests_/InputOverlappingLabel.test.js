import React from 'react';
import { render, fireEvent, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { InputOverlappingLabel } from '../components/UI/Input/InputOverlappingLabel.js';

/* const props = {
    value: "Sample Value",
    placeholder: "Enter something",
    onChange: jest.fn(),
  }; */

describe('InputOverlappingLabel', () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<InputOverlappingLabel />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  /* it("renders with provided value and placeholder", () => {
    render(<InputOverlappingLabel {...props} />);
    const inputField = screen.getByLabelText("Enter something");
    expect(inputField).toHaveValue("Sample Value");
  }); */
});