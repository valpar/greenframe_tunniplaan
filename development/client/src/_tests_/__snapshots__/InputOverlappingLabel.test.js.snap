// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputOverlappingLabel matches snapshot 1`] = `
<div
  className="relative"
>
  <label
    className="absolute left-0 ml-2 text-xs -top-2 text-gray-400 bg-transparent duration-200"
    htmlFor="input"
  />
  <input
    autoComplete="off"
    className="p-2 border border-borderGray  w-full rounded-none font-normal focus:outline-none focus:border-black"
    name=""
    onBlur={[Function]}
    onChange={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    readOnly={false}
    type="text"
    value=""
  />
</div>
`;
